integration-tests/src/test/java/oracle/weblogic/kubernetes/ItLiftAndShiftFromOnPremDomain.java

```diff
@@ -0,0 +1,481 @@
+    assertTrue(result, "Failed to create secrets");
+
+    logger.info("Run kubectl to create the domain");
+    params = new CommandParams().defaults();
+    params.command("kubectl apply -f "
+        + Paths.get(LIFT_AND_SHIFT_WORK_DIR, "/u01/", DISCOVER_DOMAIN_OUTPUT_DIR + "/" + WKO_DOMAIN_YAML).toString());
+
+    result = Command.withParams(params).execute();
+    assertTrue(result, "Failed to create domain custom resource");
+
+    // wait for the domain to exist
+    logger.info("Checking for domain custom resource in namespace {0}", domainNamespace);
+    withStandardRetryPolicy
+        .conditionEvaluationListener(
+            condition -> logger.info("Waiting for domain {0} to be created in namespace {1} "
+                    + "(elapsed time {2}ms, remaining time {3}ms)",
+                domainUid,
+                domainNamespace,
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(domainExists(domainUid, DOMAIN_VERSION, domainNamespace));
+
+    // verify the admin server service and pod is created
+    checkPodReadyAndServiceExists(adminServerPodName, domainUid, domainNamespace);
+
+    // verify managed server services created and pods are ready
+    for (int i = 1; i <= replicaCount; i++) {
+      checkPodReadyAndServiceExists(managedServerPrefix + i, domainUid, domainNamespace);
+    }
+
+    // create ingress rules with path routing for Traefik
+    createTraefikIngressRoutingRules(domainNamespace);
+
+    int traefikNodePort = getServiceNodePort(traefikNamespace, traefikHelmParams.getReleaseName(), "web");
+    assertTrue(traefikNodePort != -1,
+        "Could not get the default external service node port");
+    logger.info("Found the Traefik service nodePort {0}", traefikNodePort);
+    logger.info("The K8S_NODEPORT_HOST is {0}", K8S_NODEPORT_HOST);
+
+    String curlString = String.format("curl -v --show-error --noproxy '*' "
+            + "http://%s:%s/opdemo/?dsName=testDatasource",
+        K8S_NODEPORT_HOST, traefikNodePort);
+    
+    // check and wait for the application to be accessible in admin pod
+    checkAppIsRunning(
+        withQuickRetryPolicy,
+        domainNamespace,
+        adminServerPodName,
+        "7001",
+        "opdemo/index.jsp",
+        "WebLogic on prem to wko App");
+
+    ExecResult execResult = null;
+    logger.info("curl command {0}", curlString);
+
+    execResult = assertDoesNotThrow(
+        () -> exec(curlString, true));
+
+    if (execResult.exitValue() == 0) {
+      logger.info("\n HTTP response is \n " + execResult.stdout());
+      logger.info("curl command returned {0}", execResult.toString());
+      assertTrue(execResult.stdout().contains("WebLogic on prem to wko App"),
+          "Not able to access the application");
+    } else {
+      fail("HTTP request to the app failed" + execResult.stderr());
+    }
+  }
+
+  private static V1Pod setupWebLogicPod(String namespace) {
```

Comment:
Is it possible to use the method from BuildApplication?

Reply:
Done